### PR TITLE
refactoring of effective armour points computation

### PR DESCRIPTION
**Pull request by ghost**
displaced code from Armure to UPReference and introduced rounding via Math.round() instead of simple casts to int.

### CODE REVIEW

**Comment by ykonoclast** [comment]
add another comment

**Comment by ghost** [comment]
added comment

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /build/
-/dist/
\ No newline at end of file
+/dist/
```
Comment (ghost) [inline .gitignore:2]: don't know why this changed

Comment (ykonoclast) [inline .gitignore:2]: me neither

**Pull request merged**